Mathlib/NumberTheory/FLT/OfOddPrimes.lean

```diff
@@ -0,0 +1,25 @@
+/-
+Copyright (c) 2023 Ruben Van de Velde. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde
+-/
+
+import Mathlib.NumberTheory.FLT.Four
+
+/-!
+# Reduce Fermat's Last Theorem to odd primes
+
+To prove Fermat's Last Theorem, it suffices to prove it for odd prime exponents, and the well-known
+case of exponent 4 (`fermatLastTheoremFour`).
+-/
+
+theorem FermatLastTheoremWith.of_odd_primes
```

Comment:
Another option would be to place this result in the `FLT.Four` file, as a corollary of the main theorem.